### PR TITLE
feat(Forms): add `layoutOptions` for enhanced horizontal label layout

### PR DESCRIPTION
**Pull request by tujoworker**
This PR introduces a new property, `layoutOptions`, allowing for adjustment of the width in horizontally aligned layouts.
Since its primary use case is for input fields, we are also updating the docs examples for the [String](https://eufemia.dnb.no/uilib/extensions/forms/base-fields/String/) and [Number](https://eufemia.dnb.no/uilib/extensions/forms/base-fields/Number/) fields accordingly.

Quick example:

```tsx
<Field.Provider
  layout="horizontal"
  layoutOptions={{
    width: 'medium', // can be a rem value
  }}
>
  <Field.String label="Label text" />
  <Field.Number label="Label with a long text that will wrap" />
  <Field.Currency label="Currency field />
</Field.Provider>

```

### CODE REVIEW

File: packages/dnb-eufemia/src/style/themes/theme-sbanken/sbanken-theme-forms.scss [hunk 1]
```diff
@@ -16,4 +16,4 @@ $THEME_FALLBACK: 'ui';
 
 @import '../../../extensions/forms/Field/Number/style/themes/dnb-number-theme-sbanken.scss';
 @import '../../../extensions/forms/Wizard/style/themes/dnb-wizard-layout-theme-sbanken.scss';
-@import '../../../extensions/forms/FieldBlock/style/themes/dnb-field-block-theme-ui.scss';
+@import '../../../extensions/forms/FieldBlock/style/themes/dnb-field-block-theme-sbanken.scss';
```
Comment (langz) [inline packages/dnb-eufemia/src/style/themes/theme-sbanken/sbanken-theme-forms.scss:19]: Did this error cause some kind of styling issue as of today? 🤔

File: packages/dnb-eufemia/src/extensions/forms/Field/PhoneNumber/__tests__/__image_snapshots__/phonenumber-have-to-match-small-screen.snap.png
Comment (langz) [inline packages/dnb-eufemia/src/extensions/forms/Field/PhoneNumber/__tests__/__image_snapshots__/phonenumber-have-to-match-small-screen.snap.png:1]: I guess it makes sense that the height has decreased in this screenshot as part of this PR?

**Comment by langz** [comment]
Should we also document the props `layout` and `layoutOptions` here?
https://eufemia-git-feat-label-horizontal-layout-eufemia.vercel.app/uilib/extensions/forms/feature-fields/Provider/properties/

File: packages/dnb-eufemia/src/extensions/forms/FieldBlock/FieldBlockDocs.ts [hunk 2]
```diff
@@ -21,6 +21,16 @@ export const fieldBlockSharedProperties: PropertiesTableProps = {
     type: 'string',
     status: 'optional',
   },
+  layoutOptions: {
```
Comment (langz) [inline packages/dnb-eufemia/src/extensions/forms/FieldBlock/FieldBlockDocs.ts:24]:
```suggestion
  horizontalLayoutOptions: {
```

Based on the description of the `layoutOptions` prop, it seems like its only use is to set additional options for the `horizontal` layout. Perhaps the prop name should reflect so as well? 🤔

**Review by langz — APPROVED** [review]

File: packages/dnb-eufemia/src/style/themes/theme-sbanken/sbanken-theme-forms.scss (hunk 1 above)
Comment (tujoworker) [reply packages/dnb-eufemia/src/style/themes/theme-sbanken/sbanken-theme-forms.scss:19]: Yes, indeed.

`labelDescription` did change its color. But we did not have any visual tests on this one, so I added two now.

**Review by tujoworker — COMMENTED** [review]
Nice findings. Thank you so much. I added some more tests.

**Pull request merged**